sagemaker-autopilot/housing_prices/autopilot_california_housing.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<blockquote>on autopilot (hence the name)</blockquote><p>Since this definition is not adding any information, can you be a little more specific about what it means for to use the service on autopilot? It needs no human interaction or input?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>